tests/manage/z_cluster/cluster_expansion/test_add_capacity_entry_exit_criteria.py

```diff
@@ -31,14 +39,18 @@
 )
 @ignore_leftovers
 @tier2
-@pytest.mark.skipif(
-    config.ENV_DATA["platform"].lower() == constants.VSPHERE_PLATFORM,
-    reason=(
-        "Skipping add capacity test on vsphere cause of issue: "
-        "https://github.com/red-hat-storage/ocs-ci/issues/2356"
-    ),
-)
+@skipif_bm
+@skipif_external_mode
```

Comment:
Any specific reason to skip it over BM and Ext mode?

Reply:
I think we need to skip it with external mode because we check the osd pods. And about the BM, I am skipping it because of the comment you mentioned here: https://github.com/red-hat-storage/ocs-ci/issues/6840#issuecomment-1436777276.

Reply:
ACK!
@PrasadDesala Do you think we could support this over external mode?

Reply:
This test can be skipped on external mode